Kickstarter-iOS/UIView+AutoLayout.swift

```diff
@@ -0,0 +1,14 @@
+import UIKit
+
+extension UIView {
+  public func constrainEdges(to view: UIView) {
```

Comment:
The function naming is correct 🙂 
Good idea on the constraints, will update 👍

Reply:
Let's chat more about the constraints - one or two edge cases occurred to me. To be honest, ideally we'd solve something like this with a decent AutoLayout DSL.

Reply:
As per offline chat - we want the table view to extend all the way to the bottom because it's modally displayed. The article is referring specifically to the deprecation of `topLayoutGuide` and `bottomLayoutGuide` on `UIViewController` which we should replace with `safeAreaLayoutGuide` on iOS 11+ 👍